activerecord/lib/active_record/migration.rb

```diff
@@ -278,8 +278,10 @@ def initialize(current: nil, stored: nil)
   #
   # * <tt>change_column(table_name, column_name, type, options)</tt>:  Changes
   #   the column to a different type using the same parameters as add_column.
-  # * <tt>change_column_default(table_name, column_name, default)</tt>: Sets a
-  #   default value for +column_name+ defined by +default+ on +table_name+.
+  # * <tt>change_column_default(table_name, column_name, default_or_changes)</tt>:
+  #   Sets a default value for +column_name+ defined by +default_or_changes+ on
+  #   +table_name+. Passing a hash containing <tt>:from</tt> and <tt>:to</tt>
+  #   will make this change reversible in migration.
```

Comment:
Small nit:


Passing a hash containing <tt>:from</tt> and <tt>:to</tt> **as default_or_changes** will make this change reversible in **the** migration.